tests/helpers/acceptance.js

```diff
@@ -55,6 +58,18 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
+    .then(() => {
+      if (options.createESLintConfig) {
```

Comment:
Everything else is deferring to the upstream Ember command, this is `mock`ing it. I'd _much_ prefer (given this is an acceptance test) to actually run the generator.